### PR TITLE
ensures changed_layers is always created

### PR DESCRIPTION
**Pull request by jpbochi**

### CODE REVIEW

File: toolbox/bin/variables.sh [hunk 1]
```diff
@@ -58,7 +58,9 @@ fi
 # populate current layer info
 LAYERS_DIR="$WORKING_DIR/layers"
 LAYERS=''
-if [ -d "$LAYERS_DIR" ]; then
+if [ ! -d "$LAYERS_DIR" ]; then
```
Comment (martinb3) [inline toolbox/bin/variables.sh]: This feels weird to do on a repo that doesn't have layers. Should we add a comment to explain why this is here?

File: toolbox/bin/variables.sh [hunk 2]
```diff
@@ -58,7 +58,11 @@ fi
...
+  # CircleCI fails the build is none of the files mentioned in persist_to_workspace exist.
```
Comment (dbburgess) [inline toolbox/bin/variables.sh]:
```suggestion
  # CircleCI fails the build if none of the files mentioned in persist_to_workspace exist.
```

File: toolbox/bin/variables.sh [hunk 3]
```diff
@@ -58,7 +58,11 @@ fi
...
+  # changed_layers is the only one that makes to create in this situation
```
Comment (dbburgess) [inline toolbox/bin/variables.sh]: I'm not sure what you meant to say here?

Comment (jpbochi) [reply toolbox/bin/variables.sh]: changed phrasing there

File: toolbox/bin/variables.sh [hunk 4]
```diff
@@ -58,7 +58,11 @@ fi
 # populate current layer info
 LAYERS_DIR="$WORKING_DIR/layers"
 LAYERS=''
-if [ -d "$LAYERS_DIR" ]; then
+if [ ! -d "$LAYERS_DIR" ]; then
+  # CircleCI fails the build if none of the files mentioned in persist_to_workspace exist.
+  # changed_layers is the only one that makes to create in this situation
+  echo > "$WORKSPACE_DIR/changed_layers"
```
Comment (j00k) [inline toolbox/bin/variables.sh]: If you just touch changed_layers this wouldn't need to be conditional is that right?

**Pull request merged**